app/models/shipment.rb

```diff
@@ -1,4 +1,24 @@
+# == Schema Information
+#
+# Table name: shipments
+#
+#  id                  :integer          not null, primary key
+#  company_id          :integer
+#  destination_country :string
+#  origin_country      :string
+#  tracking_number     :string
+#  slug                :string
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#
+
 class Shipment < ApplicationRecord
   belongs_to :company
-  belongs_to :shipment_item
+  has_many :shipment_items
+
+  def grouped_shipment_items
+    shipment_items.group(:description).count.sort_by{|k, v| v}.reverse.reduce([]) do |result, (description, count)|
```

Comment:
reverse could be removed

```suggestion
    shipment_items.group(:description).count.sort_by{|k, v| -v}.reduce([]) do |result, (description, count)|
```

Reply:
Yes. Thanks for suggestion.

Reply:
do you think this should be indexed?